web-tester/src/main/java/com/axonivy/ivy/webtest/IvyWebTestExtension.java

```diff
@@ -51,7 +50,7 @@ public void beforeEach(ExtensionContext context) throws Exception {
 
   private boolean headless(ExtensionContext context) {
     return context.getConfigurationParameter("ivy.selenide.headless")
-        .map(BooleanUtils::toBoolean)
+        .map(Boolean::parseBoolean)
```

Comment:
This could possibly be a breaking change because commons-lang `BooleanUtils.toBoolean` accepts `true`, `TRUE` `tRUe`, `on`, `yes`, `y` and `t` as a `True` value. but java `Boolean.parseBoolean` only accepts the word `true` but also case insensitive

Reply:
i could also make a helper function that would make it work like before

Reply:
I'd risk to only support true/false, so keep it as is :crossed_fingers: